h2o-py/h2o/backend/connection.py

```diff
@@ -736,11 +736,13 @@ def _process_response(response, save_to):
         status_code = response.status_code
         if status_code == 200 and save_to:
             if save_to.startswith("~"): save_to = os.path.expanduser(save_to)
+            save_to = os.path.join(str(save_to), '')
```

Comment:
I think it would be more readable to do `if save_to.endswith("/"): save_to = save_to + "/"` and then mkdirs() (pardon my pseudo-python)

Reply:
`os.path.join(str(save_to), '')` has no effect if it already ends with "/" - the check is inside, this is why I omitted it here. The premature optimization inside me made me do this !

Also, they already solved the os-independency. E.g. there are some Windows users out there who use "\" instead of "/".